apps/test-stress-can/src/main.cpp

```diff
@@ -0,0 +1,38 @@
+#include "mbed.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+
+CANBus can(CAN_RX, CAN_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANMsg msg;
+
+DigitalOut led(LED1);
+InterruptIn btn(BUTTON_1);
+
+constexpr uint16_t sent = 1000;
+uint16_t received = 0;
+ 
+// create an event queue
+EventQueue queue;
+
+void receiveMessageOutside(CANMsg &msg) {
+  // process msg or something
+  received++;
+}
+ 
+void receiveMessageInside() {
+  CANMsg msg;
+  if (can.read(msg)){
```

Comment:
by definition this should always be true because the irq only fires if theres a new msg. maybe keep this as a sanity check and have some serious error here if its false